scripts/verify-snaps.ts

```diff
@@ -62,10 +69,19 @@ async function verifySnapVersion(
     );
   }
 
-  if (checksum !== manifest.source.shasum) {
+  if (registryChecksum !== manifest.source.shasum) {
     throw new Error(
       // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-      `Checksum for "${snap.id}@${version}" does not match the checksum in the registry. Expected "${manifest.source.shasum}" (manifest checksum), got "${checksum}" (registry checksum).`,
+      `Checksum for "${snap.id}@${version}" does not match the checksum in the registry. Expected "${manifest.source.shasum}" (manifest checksum), got "${registryChecksum}" (registry checksum).`,
+    );
+  }
+
+  const computedChecksum = await getSnapChecksum(fetchedSnap);
+
+  if (computedChecksum !== manifest.source.shasum) {
+    throw new Error(
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      `Checksum for "${snap.id}@${version}" does not match computed checksum. Expected "${manifest.source.shasum}" (manifest checksum), got "${computedChecksum}" (computed checksum).`,
```

Comment:
Nit: the expected value should be the computed one.

Reply:
Good point, fixed!